core/iwasm/libraries/debug-engine/debug_engine.c

```diff
@@ -54,10 +54,22 @@ allocate_instance_id()
     return id;
 }
 
+static bool
+should_run(WASMDebugControlThread *control_thread)
```

Comment:
how about rename to `is_running`?

Reply:
Had better rename to `is_thread_running`, `is_thread_stopped`, and `is_thread_detached` to make the code more clear.